server/src/test/java/io/crate/analyze/OptimizeTableAnalyzerTest.java

```diff
@@ -71,111 +75,114 @@ private OptimizeTablePlan.BoundOptimizeTable analyze(String stmt, Object... argu
     }
 
     @Test
-    public void testOptimizeSystemTable() throws Exception {
+    public void testOptimizeSystemTable() {
```

Comment:
Please don't remove `throws Exception` clauses - it's noisy and annoying to re-add them once a test starts calling a checked exception.

If this is a Intellij linting warning we should disable it. it's not an "improvement" to remove these for tests.